ci/build-binaries-and-appimage.sh

```diff
@@ -40,11 +40,20 @@ OLD_CWD="$(readlink -f .)"
 pushd "$BUILD_DIR"
 
 # configure build and generate build files
-cmake "$REPO_ROOT" \
-    -DCMAKE_INSTALL_PREFIX=/usr \
+
+CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=/usr \
     -DCMAKE_BUILD_TYPE=RelWithDebInfo \
     -DBUILD_TESTING=ON \
-    -DAPPIMAGEKIT_PACKAGE_DEBS=ON
+    -DAPPIMAGEKIT_PACKAGE_DEBS=ON"
+
+if [[ $ARCH =~ loongarch ]]
+then
```

Comment:
Multiple syntax/code style issues. We put `then` on the same line. `CMAKE_ARGS` should be a true bash array if used like this. The `$ARCH` variable is not escaped properly.

Use `shellcheck` to make sure your code works.

Reply:
done.